sc-locked-asset-factory/src/locked_asset.rs

```diff
@@ -175,9 +177,7 @@ pub trait LockedAssetModule: asset::AssetModule {
             percents_sum += milestone.unlock_percent;
         }
 
-        if !unlock_milestones.is_empty() {
-            require!(percents_sum == 100, "Percents do not sum up to 100");
-        }
+        require!(percents_sum == 100, "Percents do not sum up to 100");
```

Comment:
Extract the "100" as a PERCENTAGE_TOTAL constant or similar.

Reply:
Done